frontend/src/metabase/redux/metadata.js

```diff
@@ -313,11 +313,10 @@ export const loadMetadataForQuery = (query, extraDependencies) =>
   loadMetadataForQueries([query], extraDependencies);
 
 export const loadMetadataForQueries =
-  (queries, extraDependencies = []) =>
-  dispatch => {
+  (queries, extraDependencies, options) => dispatch => {
     const dependencies = _.chain(queries)
       .map(q => q.dependentMetadata())
-      .push(...extraDependencies)
+      .push(...(extraDependencies ?? []))
```

Comment:
Is there any particular reason you moved the default value from the parameter here? I guess you were trying to improve the parameter readability?

Reply:
No, it's not allowed to use default parameters for parameters that are not last ones.